src/cfml/system/services/ServerService.cfc

```diff
@@ -703,6 +703,21 @@ component accessors="true" singleton {
 		serverJSON.trayOptions = serverJSON.trayOptions.map( function( item ){
 			if( item.keyExists( 'image' ) && item.image.len() ) {
 				item.image = fileSystemUtil.resolvePath( item.image, defaultServerConfigFileDirectory );
+			}else{
```

Comment:
@mgmathus Why is this in an `else`? We want this logic to run regardless of wether the image logic above runs.